public/content/documentation/web/component/autocomplete.md

```diff
@@ -0,0 +1,156 @@
+## General Notes
+
+How to test an autocomplete input with listbox
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab: Focus moves visibly to the text input
+   - Arrow-keys: Focus moves to and selects the next option. If the textbox is empty and the listbox is not displayed, opens the listbox and moves visual focus to the next option. In both cases DOM focus remains on the textbox.
+   - Enter: The textbox value is set to the content of the selected option. The listbox closes.
+   - Escape: Clears the textbox. If the listbox is displayed, closes it.
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the input, traverses list
+   - Doubletap: Opens select, chooses option
+
+3. Listen to screenreader output on all devices
+
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a select, popup, menu/submenu, listbox or combobox
+   - Group: Its label is read and selected options are read
+   - State: It indicates the value of the text input
+     
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+GIVEN THAT I am on a page with a header landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to the text input I SEE focus is strongly visually indicated
+   - THEN when I use the arrow keys to select an option I SEE the selected option is the new text input value
+   - THEN when I use the enter key I SEE the selected option is changed and focus returns to the text input
+   - THEN when I use the escape key when the select is open I SEE it collapses and focus moves to the text input
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND I use the tab key to move focus to the text input
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a select, popup, menu/submenu, listbox or combobox
+      - I HEAR Its label is read and selected options are read
+      - I HEAR It indicates the value of the text input
+   - THEN when I use the arrow keys to select an option I HEAR the selected option is the new text input value
+   - THEN when I use the enter key I HEAR the selected option is changed and focus returns to the text input
+   - THEN when I use the escape key when the select is open I HEAR it collapses and focus moves to the text input
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND I swipe to focus on a select
+      - I HEAR Its purpose is clear
```

Comment:
Same here. After each "I HEAR" on this line and the three below, all of the first letters of the next words are capitalized. Let's lowercase them ("I" in "Its" twice, and "I" in "It" twice).